lib/index.js

```diff
@@ -10,12 +10,16 @@ function inspect(basePath, fileName) {
   try {
     // lockfile. usually composer.lock
     composerLockObj = loadJsonFile(basePath, fileName);
+    // throw an error if .json file was specified
+    if (typeof composerLockObj.packages !== 'object') {
+      throw new Error('Please specify the lock file name');
```

Comment:
The error message should not be about the corrective action (it assumes more about the error that it should), but about the failure. For example, `new Error('Invalid lock file ' + basePath + '/' + fileName)`